bot/exts/utilities/color.py

```diff
@@ -0,0 +1,229 @@
+import colorsys
+import json
+import pathlib
+import random
+from io import BytesIO
+
+from PIL import Image, ImageColor
+from discord import Embed, File
+from discord.ext import commands
+from discord.ext.commands.errors import BadArgument
+from rapidfuzz import process
+
+from bot.bot import Bot
+from bot.exts.core.extensions import invoke_help_command
+
+THUMBNAIL_SIZE = (80, 80)
+
+
+class Colour(commands.Cog):
+    """Cog for the Colour command."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        with open(pathlib.Path("bot/resources/utilities/ryanzec_colours.json")) as f:
+            self.colour_mapping = json.load(f)
+
+    async def send_colour_response(self, ctx: commands.Context, rgb: tuple[int, int, int]) -> None:
+        """Create and send embed from user given colour information."""
+        r, g, b = rgb
+        name = self._rgb_to_name(rgb)
+        if name == "No match found":
+            name = None
```

Comment:
Why not just make `self._rgb_to_name` return `None`?

Reply:
Because it doesn't look good when it is on the top line of the embed vs the value for the Name field.

Reply:
I didn't understand, we are only using `self._rgb_to_name` once in the full source and when it returns `No match found` we make it `None`.

Reply:
> 
> 
> Because it doesn't look good when it is on the top line of the embed vs the value for the Name field.

Rather than convert No match found to none, what about converting None to No match found?
This means that the code is consistent and the string 'No match found' only has to be in the code in one place, rather than several.

Reply:
Fixed with [c869faf](https://github.com/python-discord/sir-lancebot/pull/842/commits/c869faf32ceeadc94a8a38b5370706a4b6da32ae)